packages/worker/src/api/controllers/global/users.ts

```diff
@@ -328,33 +306,31 @@ export const invite = async (ctx: Ctx<InviteUserRequest>) => {
 }
 
 export const inviteMultiple = async (ctx: Ctx<InviteUsersRequest>) => {
-  const request = ctx.request.body
-  ctx.body = await userSdk.invite(request)
+  ctx.body = await userSdk.invite(ctx.request.body)
 }
 
 export const checkInvite = async (ctx: any) => {
   const { code } = ctx.params
   let invite
   try {
-    invite = await checkInviteCode(code, false)
```

Comment:
I renamed this function to `getInviteCode` to better reflect what it actually does, and removed the functionality within it to delete tokens. It felt dangerous for a function called `checkInviteCode` to, unless you pass it `false` as a second argument, delete data. There's only one place in the codebase where this delete functionality is used, and it now calls a delete function directly.